pylint_odoo/misc.py

```diff
@@ -112,14 +113,27 @@ def get_manifest_file(self, node_file):
 
     def set_ext_files(self):
         """Create `self.ext_files` dictionary with {extension_file: [files]}
+            and exclude files using --ignore and --ignore-patterns parameters
         """
         self.ext_files = {}
+        ignores = list(self.linter.config.black_list)
```

Comment:
The variable is not necessary since it is used only once!
This change had been made by the https://github.com/Vauxoo/pylint-odoo/pull/103/commits/188d405599debc325f4dd3be2f12d28ef44faf67 commit excluding the test folder
Can I return this change?